test/e2e/node_feature_discovery.go

```diff
@@ -45,7 +45,7 @@ import (
 
 var (
 	dockerRepo    = flag.String("nfd.repo", "k8s.gcr.io/nfd/node-feature-discovery", "Docker repository to fetch image from")
-	dockerTag     = flag.String("nfd.tag", "v0.6.0", "Docker tag to use")
+	dockerTag     = flag.String("nfd.tag", "master", "Docker tag to use")
```

Comment:
You need to change `nfd.repo`, too as the production repo `k8s.gcr.io/nfd/node-feature-discovery`  does not have `master` tag

Reply:
oh staging .... sorry missed that part